enableEpochs.toml

```diff
@@ -1,370 +1,363 @@
 
     # ScToScLogEventEnableEpoch represents the epoch when the sc to sc log event feature is enabled
-    ScToScLogEventEnableEpoch = 1265
+    ScToScLogEventEnableEpoch = 0
 
     # NFTStopCreateEnableEpoch represents the epoch when NFT stop create feature is enabled
-    NFTStopCreateEnableEpoch = 1265
+    NFTStopCreateEnableEpoch = 0
 
     # ChangeOwnerAddressCrossShardThroughSCEnableEpoch represents the epoch when the change owner address built in function will work also through a smart contract call cross shard
-    ChangeOwnerAddressCrossShardThroughSCEnableEpoch = 1265
+    ChangeOwnerAddressCrossShardThroughSCEnableEpoch = 0
 
     # FixGasRemainingForSaveKeyValueBuiltinFunctionEnableEpoch represents the epoch when the fix for the remaining gas in the SaveKeyValue builtin function is enabled
-    FixGasRemainingForSaveKeyValueBuiltinFunctionEnableEpoch = 1265
+    FixGasRemainingForSaveKeyValueBuiltinFunctionEnableEpoch = 0
 
     # CurrentRandomnessOnSortingEnableEpoch represents the epoch when the current randomness on sorting is enabled
-    CurrentRandomnessOnSortingEnableEpoch = 1391
+    CurrentRandomnessOnSortingEnableEpoch = 0
 
     # StakeLimitsEnableEpoch represents the epoch when stake limits on validators are enabled
     # Should have the same value as StakingV4Step1EnableEpoch that triggers the automatic unstake operations for the queue nodes
-    StakeLimitsEnableEpoch = 1391
+    StakeLimitsEnableEpoch = 0
 
     # StakingV4Step1EnableEpoch represents the epoch when staking v4 is initialized. This is the epoch in which
     # all nodes from staking queue are moved in the auction list
-    StakingV4Step1EnableEpoch = 1391
+    StakingV4Step1EnableEpoch = 0
 
     # StakingV4Step2EnableEpoch represents the epoch when staking v4 is enabled. Should have a greater value than StakingV4Step1EnableEpoch.
     # From this epoch, all shuffled out nodes are moved to auction nodes. No auction nodes selection is done yet.
-    StakingV4Step2EnableEpoch = 1392
+    StakingV4Step2EnableEpoch = 0
 
     # StakingV4Step3EnableEpoch represents the epoch in which selected nodes from auction will be distributed to waiting list
-    StakingV4Step3EnableEpoch = 1393
-
-    # AlwaysMergeContextsInEEIEnableEpoch represents the epoch in which the EEI will always merge the contexts
-    AlwaysMergeContextsInEEIEnableEpoch = 1391
+    StakingV4Step3EnableEpoch = 0
 
     # CleanupAuctionOnLowWaitingListEnableEpoch represents the epoch when duplicated data cleanup from auction list is enabled in the condition of a low waiting list
     # Should have the same value as StakingV4Step1EnableEpoch if the low waiting list has not happened, otherwise should have a greater value
-    CleanupAuctionOnLowWaitingListEnableEpoch = 1391
+    CleanupAuctionOnLowWaitingListEnableEpoch = 0
+
+    # AlwaysMergeContextsInEEIEnableEpoch represents the epoch in which the EEI will always merge the contexts
+    AlwaysMergeContextsInEEIEnableEpoch = 0
```

Comment:
Maybe move this above `CleanupAuctionOnLowWaitingListEnableEpoch`

Reply:
Why?

Reply:
That's the order in [mainnet configs](https://github.com/multiversx/mx-chain-mainnet-config/blob/3ed1c86588b8608b3ddb57f5c235f9f77418ca2d/enableEpochs.toml#L310)